docs/threatmodel.md

```
@@ -0,0 +1,143 @@
+```bash
+# Single provider
+pipeline_check --pipeline gitlab --gitlab-path .gitlab-ci.yml \
+    --output threatmodel --output-file threatmodel.md
+
+# Multi-provider (cross-provider chains land in the document too)
+pipeline_check --pipelines github,oci --output threatmodel
+```
+
+## Document layout
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)